README.md

```diff
@@ -468,4 +469,138 @@ class CalculateUserTransactionMetricsTestCase(TestCase):
 
 if __name__ == '__main__':
     main()
-```
\ No newline at end of file
+```
+
+### Creating beam manager
+Beam manager is an object that allows you to create dataflow pipelines using `create_dataflow_pipeline` method and 
```

Comment:
A beam manager object allows you to create a dataflow pipeline. The `create_dataflow_pipeline` function allows you to create the beam manager object. Beam manager provides utility methods, wrapping raw beam API: `write_truncate_to_big_query`, `write_to_avro`, `read_from_big_query`, `read_from_avro`.

Reply:
Basing on your suggestion I've made changes